Packs/SafeBreach/Integrations/SafeBreach_v2/SafeBreach_v2_description.md

```diff
@@ -2,11 +2,11 @@
 This integration leverages SafeBreach simulation results and insights to remediate malicious indicators that expose your environment to real risks.
 
   To configure the integration on SafeBreach:
-  1. Open the **Navigation bar** → … → **CLI Console**
-  2. Type **config accounts** to find out the account id
-  3. Use the id as the **accountId** parameter in Demisto configuration
+  1. Open the **Navigation bar** → … → **CLI Console**.
+  2. Type **config accounts** to get the account id.
+  3. Use the id as the **accountId** parameter when configuring the SafeBreach integration in Demisto.
   4. Type **config apikeys** to list existing API keys \
   OR \
   Add a new one by typing: **config apikeys add --name <key_name>**
-  5. Use the generated API token as **apiKey** parameter in Demisto configuration
-  6. Use your SafeBreach Management URL as the **url** parameter in Demisto configuration
\ No newline at end of file
+  5. Use the generated API token as **apiKey** parameter when configuring the SafeBreach integration in Demisto.
+  6. Use your SafeBreach Management URL as the **url** parameter when configuring the SafeBreach integration in Demisto.ion.
```

Comment:
@kirbles19 
typo? "Demisto.ion"?

Reply:
probably.
Please change it to `Demisto.`